fastlane-plugin-trainer/lib/fastlane/plugin/trainer/actions/trainer_action.rb

```diff
@@ -4,7 +4,7 @@ class TrainerAction < Action
       def self.run(params)
         require "trainer"
 
-        params[:path] = Actions.lane_context[Actions::SharedValues::SCAN_GENERATED_PLIST_FILE] if Actions.lane_context[Actions::SharedValues::SCAN_GENERATED_PLIST_FILE]
+        params[:path] ||= Actions.lane_context[Actions::SharedValues::SCAN_GENERATED_PLIST_FILE] if Actions.lane_context[Actions::SharedValues::SCAN_GENERATED_PLIST_FILE]
```

Comment:
If you weren't previously setting 'path' it will now always be '.' (the options default) even if SCAN_GENERATED_PLIST_FILE exists.  If you want everything to be passive you could do something like [this](https://github.com/lyricsboy/trainer/compare/lyricsboy/multiple-test-summaries...CraigLn:lyricsboy/multiple-test-summaries).  I'm not sure what the best fix is for people using with/without Fastlane, since they're both assuming different default values.

Looks good; thanks for creating the PR.